src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private static final int FULLNAME_INDEX = 0;
+    private static final int AGE_INDEX = 1;
+    private static final int VALUE_INDEX = 2;
+    private static final int RISK_INDEX = 3;
+    @Inject
+    private BetDao betDao;
+    @Inject
+    private UserDao userDao;
+
+    public void handle() {
+
```

Comment:
```suggestion
```
I agree - let's not add empty lines after method signature

Reply:
done